homeassistant/components/aftership/sensor.py

```diff
@@ -94,7 +149,13 @@ def icon(self):
         """Icon to use in the frontend."""
         return ICON
 
+    async def async_added_to_hass(self):
+        """Register callbacks."""
+        self.hass.helpers.dispatcher.async_dispatcher_connect(
+            UPDATE_TOPIC, self.async_update)
```

Comment:
I think we need to make a new method, as signal target, that makes the forced update call and then schedules a state update.

```py
async def force_update(self):
    await self.async_update(no_throttle=True)
    await self.async_update_ha_state()
```